contracts/libraries/SafeToL2Migration.sol

```diff
@@ -110,8 +106,9 @@ contract SafeToL2Migration is SafeStorage {
      * Singletons versions will be compared, so it implies that contracts exist
      */
     function migrateToL2(address l2Singleton) public onlyDelegateCall onlyNonceZero {
```

Comment:
I'm surprised they didn't catch this, but some of these functions should be `external` and not `public` right?

Reply:
Yes, updated 👍🏾